Contributing.md

```diff
@@ -0,0 +1,97 @@
+* With them all committed, push them, this pushes everything in that branch up. 
+
+### Submitting Changes
+* A Pull Request is the way to notify the project maintainers that you have some work that they should review and add to the project. 
+* You’re requesting that they pull your changes in. Read the article ["Using Pull Requests"](https://help.github.com/articles/using-pull-requests) on GitHub.
+* Then you can go back to your forked branch and issue a pull request to the original repository and choose the correct original branch (should be **`develop`**).
+* To create one, go to your fork of the project, click on the Pull Requests tab, and click the big green “New Pull Request” button.
+
+### Merging Changes
+**We want your help to make CARMAPlatform Project great.**
+* All pull requests will be reviewed by the CARMAPlatform team. 
+* During the review of your pull request the team member will either merge it, request changes to it, or close it with an explanation. 
+* For major changes the reviewer may require additional support from the team, which could cause some delay. 
+* We'll do our best to provide updates and feedback throughout the process. 
+* Feel free to open pull requests, and the CARMAPlatform team will communicate through it with any comments.
+
+## Branching Model
+In order to make the development process efficient, please comply with the branching model described below. On this model, we mainly use five branches - master, develop, feature, release, and hotfix.
+
+![A successful Git branching model](docs/image/Git_Workflow.png)
+
+### master
+* This branch contains production-ready code of CARMAPlatform.
+* This branch should be kept stable at all times
+
+### develop
+* This branch contains code for the upcoming release.
+* This branch should be kept stable at all times. This is important because new branches are created off of this branch.
+
+### feature
+* These branches are created from develop; **each feature branch is used to implement a single task.** After completing the task, the feature branch must be merged into develop.
+* To avoid ambiguity, we use a specific naming convention for feature branches − they always begin with feature/ followed by a description based on the functionality implemented by the feature. For example, **feature/sign-up-with-email-and-password.**
+* Thanks to this naming, team members can easily tell what code each branch contains.
+
+### release
+* This branch is created every iteration (i.e. sprint) from develop, and when the team rolls out a release it’s deployed to the staging server for testing. A stable release is merged first into the master branch and then into develop.

```

Comment:
We don't have a staging server.  And we can't guarantee that we will have a release after every sprint.  so I suggest replacing this description with:
A release branch is created whenever a set of functionality is ready for production release. It is normally created from develop and is used to keep the contents stable for final acceptance testing while other development activities may continue on other branches. Once the release is accepted, it will be merged into the master branch and versioned, then merged into develop.